### PR TITLE
[REVIEW]Fix query slowdown due to jvm contention on workers

### PR DESCRIPTION
**Pull request by VibhuJawa**
This PR fixes query slowdown due to jvm contention on workers . 

CC: @randerzander

### CODE REVIEW

File: gpu_bdb/benchmark_runner.py [hunk 1]
```diff
@@ -47,6 +57,11 @@ def load_query(qnum, fn):
 
     client, c = attach_to_cluster(config, create_sql_context=include_sql)
     # Preload required libraries for queries on all workers
+    import dask_sql
+    dask_sql_modules = [m for m in sys.modules if m.startswith('dask_sql')]
```
Comment (ayushdg) [inline gpu_bdb/benchmark_runner.py:61]: missing sys import at the top `import sys`

File: gpu_bdb/benchmark_runner.py [hunk 2]
```diff
@@ -47,6 +57,11 @@ def load_query(qnum, fn):
...
+    client.run(prevent_sql_import, dask_sql_modules)
+    print("is_jvm_started", client.run(is_jvm_started))
```
Comment (ayushdg) [inline gpu_bdb/benchmark_runner.py:63]: Can this section be moved under the dask_sql runs under `if include_sql`

**Review by ayushdg — CHANGES_REQUESTED** [review]
Minor suggestions

File: gpu_bdb/benchmark_runner.py (hunk 2 above)
Comment (VibhuJawa) [reply gpu_bdb/benchmark_runner.py:64]: I was worried that we might import `dask-sql` with `client.run(import_query_libs)` , cause those have `dask-sql` dependency's . 

I would like to keep this as is unless we have strong reasons to move it.

Comment (ayushdg) [reply gpu_bdb/benchmark_runner.py:64]: In that case could we instead move the import dask_sql into a try catch block. The idea is we don't want users who built and environment from the non dask sql yaml file to run into issues if they aren't looking to run sql queries

Comment (VibhuJawa) [reply gpu_bdb/benchmark_runner.py:64]: Ahh, gotcha. That we can do.  Let me update with that in mind.

Comment (VibhuJawa) [reply gpu_bdb/benchmark_runner.py]: Resolved by https://github.com/VibhuJawa/gpu-bdb/commit/d5ed12f40734bb7f5968051e5e661ead559062c0

File: gpu_bdb/benchmark_runner.py [hunk 3]
```diff
@@ -47,6 +58,16 @@ def load_query(qnum, fn):
 
     client, c = attach_to_cluster(config, create_sql_context=include_sql)
     # Preload required libraries for queries on all workers
+    try:
+      import dask_sql
+    except ImportError:
+      dask_sql = None
+    
+    if dask_sql is not None:
```
Comment (ayushdg) [inline gpu_bdb/benchmark_runner.py:66]: can replace with `if dask_sql`

**Review by ayushdg — APPROVED** [review]
Minor suggestion, the rest LGTM!

**Pull request merged**